doc/src/contributing/new-contributors-guide/dev-setup.md

```diff
@@ -23,7 +23,7 @@ second contribution.
 ## Install Git
 
 SymPy is available on [GitHub](https://github.com/sympy/sympy) and uses
-[Git](https://git-scm.com) for source control. The workflow is such that
+[Git](https://git-scm.com/) for source control. The workflow is such that
```

Comment:
Seems unnecessary, the older link worked just fine.

Reply:
Should I remove this?